src/igvfd/schemas/human_genomic_variant.json

```diff
@@ -0,0 +1,96 @@
+{
+    "title": "Human Genomic Variant",
+    "description": "Schema for submitting a human genomic variant.",
+    "$id": "/profiles/human_genomic_variant.json",
+    "$schema": "https://json-schema.org/draft/2020-12/schema",
+    "type": "object",
+    "required": [
+        "refseq_id",
+        "ref",
+        "alt",
+        "position",
+        "assembly"
+    ],
+    "identifyingProperties": [
+        "uuid",
+        "aliases"
+    ],
+    "additionalProperties": false,
+    "mixinProperties": [
+        {
+            "$ref": "variant.json#/properties"
+        },
+        {
+            "$ref": "mixins.json#/basic_item"
+        },
+        {
+            "$ref": "mixins.json#/standard_status"
+        }
+    ],
+    "properties": {
+        "schema_version": {
+            "default": "1"
+        },
+        "rsid": {
+            "title": "Variant ID",
+            "description": "An external ID for the variant from dbSNP.",
+            "type": "string",
+            "pattern": "rs[0-9]+"
+        },
+        "assembly": {
+            "enum": [
+                "GRCh38"
+            ]
+        },
+        "chromosome": {
+            "title": "Chromosome",
+            "description": "The number (or letter) designation for the chromosome, e.g. chr1 or chrX. May also specify the variant is in the mitochondrial DNA (mDNA).",
+            "type": "string",
+            "enum": [
+                "chr1",
+                "chr2",
+                "chr3",
+                "chr4",
+                "chr5",
+                "chr6",
+                "chr7",
+                "chr8",
+                "chr9",
+                "chr10",
+                "chr11",
+                "chr12",
+                "chr13",
+                "chr14",
+                "chr15",
+                "chr16",
+                "chr17",
+                "chr18",
+                "chr19",
+                "chr20",
+                "chr21",
+                "chr22",
+                "chrX",
+                "chrY",
+                "mDNA"
+            ]
+        },
+        "refseq_id": {
+            "title": "RefSeq Identifier",
+            "description": "The RefSeq identifier for this variant. If there is no RefSeq identifier available for this variant, the seqeunce should be used instead.",
```

Comment:
RefSeq is not a variant identifier.
It is identifier of reference sequence. please correct the description accordingly